packages/slice-machine/pages/changes.tsx

```diff
@@ -32,6 +35,27 @@ const Changes: React.FunctionComponent = () => {
     authStatus,
     isOnline,
   } = useUnSyncChanges();
+
+  type ChangedSlice = { status: ChangesStatus; c: ComponentUI };
+  type ChangedCustomType = { status: ChangesStatus; c: CustomTypeSM };
+  const { changedSlices, changedCustomTypes } = useMemo(() => {
+    const changedSlices = unSyncedSlices
+      .map((s) => ({
+        c: s,
+        status: modelsStatuses.slices[s.model.id],
+      }))
+      .filter((s): s is ChangedSlice => unSyncStatuses.includes(s.status)); // FIXME can we sync unSyncStatuses and ChangedSlice?
```

Comment:
Did you look into this one further?

Reply:
No, good catch. I only searched for _TODO_ 😅

Reply:
I'm not ignoring this btw. Spent some time trying to improve it but couldn't find anything better. Have changed this to a TODO since we also spoke about coming back to how we pass model status around in general, if that is the case later then we will re-write this anyway